lib/src/modules/conditionals/flop.dart

```diff
@@ -146,7 +152,9 @@ class FlipFlop extends Module with SystemVerilog {
     var contents = [q < _d];
 
     if (_en != null) {
-      contents = [If(_en!, then: contents)];
+      contents = [
+        If(_en!, then: contents, ifLabel: label == null ? null : '${label}_en')
```

Comment:
since `flop` uses custom `instantiationVerilog`, i think this label addition here actually has no effect.  there's no begin/end blocks at all for this one-liner in generated SV.